### PR TITLE
Adds customizable favicons

### PR DESCRIPTION
**Pull request by sdwolfz**
### Prerequisites

Put an `x` into the box(es) that apply:

- [x] This pull request fixes a bug.
- [x] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

### Description

Makes favicons customizable, since there are many possibilities for them: https://stackoverflow.com/a/26768184

### CODE REVIEW

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -80,8 +80,9 @@
       {{ end }}
     {{ end }}
 
-    <link rel="icon" type="image/png" href="{{ .Site.Params.favicon_32 | default "/images/favicon-32x32.png" | absURL }}" sizes="32x32">
-    <link rel="icon" type="image/png" href="{{ .Site.Params.favicon_16 | default "/images/favicon-16x16.png" | absURL }}" sizes="16x16">
+    {{ range .Site.Params.favicons }}
+      <link rel="{{ .rel }}" type="{{ .type }}" href="{{ .href | absURL }}" sizes="{{ .sizes }}">
```
Comment (metbril) [inline layouts/_default/baseof.html:84]: This change causes that favicons is a required config item? In my opinion the template should provide sensible defaults that can be overwritten with an optional parameter.

Comment (metbril) [reply layouts/_default/baseof.html:84]: It also introduces a **breaking change**, since the `favicons` is a new config parameter.

Comment (sdwolfz) [reply layouts/_default/baseof.html:84]: How about default the initial two, else if custom settings are specified, my changes?

**Comment by luizdepra** [comment]
Hey, this is cool but I think this theme already has too many configurations options that could be made simply by change a file or two.
You change the favicons by replacing the default ones. Also, you could add more icons by modifying `baseof.html`.

**Comment by sdwolfz** [comment]
@luizdepra OK I'll keep these changes on my fork.

**Pull request closed without merging**